src/ol/webgl/ShaderBuilder.js

```diff
@@ -548,6 +548,7 @@ export function parseLiteralStyle(style) {
 
   if (symbStyle.symbolType === 'image' && symbStyle.src) {
     const texture = new Image();
+    texture.crossOrigin = symbStyle.crossOrigin || 'anonymous';
```

Comment:
```suggestion
    texture.crossOrigin = symbStyle.crossOrigin === undefined ? 'anonymous' : symbolStyle.crossOrigin;
```
In case a user demands `''` as `crossOrigin` value.

Reply:
[My understanding](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin) is that `''` is interpreted as `'anonymous'`.